montblanc/impl/rime/tensorflow/rime_ops/radec_to_lm_op_cpu.cpp

```diff
@@ -10,7 +10,9 @@ using tensorflow::shape_inference::ShapeHandle;
 using tensorflow::shape_inference::DimensionHandle;
 using tensorflow::Status;
 
-auto shape_function = [](InferenceContext* c) {
+#include <iostream>
+//auto shape_function = [](InferenceContext* c) {
+auto shape_function(InferenceContext* c) {
```

Comment:
The return type should probably be explicitly set to Status.

Reply:
sorry this was debugging - I should revert this

Reply:
I think `Status shape_function(InferenceContext* c) {` would be more explicit.